services/portfolios_ca/src/routes/targets.ts

```diff
@@ -0,0 +1,86 @@
+import { Router } from 'restify-router';
+import { EHFunction } from '../types/config';
+import { TargetsServiceI } from '../types/targets';
+
+export default (errorHandler: EHFunction, TargetsService: TargetsServiceI) => {
+  const router = new Router();
+
+  /**
+   * @apiGroup s_portfolios_ca
+   * @api {get} /portfolios-ca/targets/values PortfoliosByTarget
```

Comment:
I think this is the case were we could use path params, similar to `SCIHFPALayer`, to keep it explicit it could be `/portfolios-ca/targets/:target_id/values` and the purpose of that endpoint would be clearer .

What do you think?

Reply:
Done. Now it is used a path param targetId